turbinia/api/routes/request.py

```diff
@@ -47,10 +48,12 @@ async def get_requests_summary(request: Request):
     if not requests_summary.get_requests_summary():
       return JSONResponse(
           content={'detail': 'Request summary is empty'}, status_code=200)
-    response_json = requests_summary.json(sort_keys=True)
+    response_json = requests_summary.model_dump_json()
+    response = json.loads(response_json)
+    if isinstance(response, dict):
```

Comment:
I think ```response_json``` will always be a dictionary in this scenario. Maybe try to catch ```JSONDecodeError```  from ```json.loads``` instead?

Reply:
Thanks, I removed the if statement and added AttributeError to the exception block beneath it.